web/src/settings_notifications.ts

```diff
@@ -211,6 +221,56 @@ function stream_notification_setting_changed(target: HTMLInputElement): void {
     );
 }
 
+function change_state_of_customize_stream_notifications_widget(
+    event: JQuery.ClickEvent,
+    dropdown: tippy.Instance,
+): void {
+    dropdown.hide();
+    event.preventDefault();
+    event.stopPropagation();
+    assert(customize_stream_notifications_widget !== undefined);
+    customize_stream_notifications_widget.render();
+    const stream_id = customize_stream_notifications_widget.value();
+    assert(typeof stream_id === "number");
+
+    const $customizable_stream_notifications_table = $("#customizable_stream_notifications_table");
+
+    $customizable_stream_notifications_table.find("input[type='checkbox']").prop("disabled", false);
```

Comment:
I think you can use blank lines better in this area to make it more readable. I would probably be doing a blank line after `stopPropagation`, but not one just before this line.